kbl2vec/src/main/java/com/foursoft/harness/kbl2vec/transform/components/terminals/TerminalDocumentVersionTransformer.java

```diff
@@ -54,6 +51,8 @@ public TransformationResult<VecDocumentVersion> transform(final TransformationCo
                                     VecDocumentVersion::getSpecifications)
                     .withDownstream(KblGeneralTerminal.class, VecTerminalReceptionSpecification.class, Query.of(source),
                                     VecDocumentVersion::getSpecifications)
+                    .withDownstream(KblGeneralTerminal.class, VecPlaceableElementSpecification.class, Query.of(source),
+                                    VecDocumentVersion::getSpecifications)
```

Comment:
This is where it gets messy in KBL. "Regular Terminals" are not placeable, therefore they should not get a PlaceableElementSpecification. However, Special Terminals are placeable (thats part of what makes them "special"). The problem is, that on a GeneralTerminal, you can't detect (in vanilla KBL) if it is a special or a regular terminal. So, here you should check if there are any SpecialTerminalOccurrences, that reference this GeneralTerminal. If not, it will not get a PlaceableElementSpecification.